docker/run-mysql.sh

```diff
@@ -13,6 +13,7 @@ start() {
     echo "starting up mysql container..."
     docker run --name $CONTAINER \
                -e MYSQL_ROOT_PASSWORD=leonardo-test \
+               -e 'MYSQL_ROOT_HOST=%' \
```

Comment:
[Here's the docs](https://hub.docker.com/r/mysql/mysql-server/) on that configuration var.

The TLDR of it is that, by default, the mysql container only allows connections coming from localhost. In my installation of docker, even the internal connections are coming from a different IP. This fix allows root connections from any host. It will fix me (and any potential builders that experience this) and it won't break anybody else.

As far as root cause, I haven't dug that deep, but It seems to be an issue with Google's default config since I've experienced the issue previously on my other corp computer.